Woof/Woof/View/SitterFlow/SitterProfileView/SitterProfileView.swift

```diff
@@ -5,105 +5,132 @@ struct SitterProfileView: View {
-            }
 
-            Spacer()
-        }
-        .padding(.horizontal)
-        .overlay(
-            Group {
-                if viewModel.isSavingData {
-                    Color.white.opacity(AppStyle.UIElementConstant.opacityLevelForProgressViewBackground)
-                    ProgressView()
-                        .progressViewStyle(CircularProgressViewStyle())
-                }
+                Spacer()
             }
-        )
-        .alert(
-            alertTitle,
-            isPresented: $viewModel.isErrorOccurred,
-            actions: {
-                Button(cancelButtonLabelText) {
-                    viewModel.cancelEditing()
+            .padding(.horizontal)
+            .buttonStyle(CapsuleWithWhiteText())
+            .overlay(
+                Group {
+                    if viewModel.isSavingData {
+                        Color.white.opacity(AppStyle.UIElementConstant.opacityLevelForProgressViewBackground)
+                        ProgressView()
+                            .progressViewStyle(CircularProgressViewStyle())
+                    }
                 }
-                Button(tryAgainButtonLabelText) {
-                    Task {
-                        await viewModel.save()
+            )
+            .alert(
+                alertTitle,
+                isPresented: $viewModel.isErrorOccurred,
+                actions: {
+                    Button(cancelButtonLabelText) {
+                        viewModel.cancelEditing()
                     }
+                    Button(tryAgainButtonLabelText) {
+                        Task {
+                            await viewModel.save()
+                        }
+                    }
+                },
+                message: {
+                    Text(viewModel.errorMessage)
+                }
+            )
+            .navigationTitle(navigationTitle)
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                Button(logoutButtonLabelText) {
+                    viewModel.isAlertShown.toggle()
+                }
+                .alert(alertLogOutTitle, isPresented: $viewModel.isAlertShown) {
+                    Button(continueButtonLabelText) {
+                        viewModel.isLogoutConfirmed.toggle()
+                        userRoleViewModel.resetCurrentRole()
+                    }
+                    Button(
+                        cancelButtonLabelText,
+                        role: .cancel
+                    ) { viewModel.isAlertShown.toggle() }
                 }
-            },
-            message: {
-                Text(viewModel.errorMessage)
             }
-        )
+        }
     }
 
     // MARK: - Private interface
 
     /// View model responsible to manage data from model layer.
     @StateObject private var viewModel = SitterProfileViewModel()
+
+    @EnvironmentObject private var userRoleViewModel: UserRoleViewModel
+    @Environment(\.dismiss) private var dismiss
```

Comment:
How do we intend to use this?

Reply:
I can still see this line in the same place.

Reply:
Which line do you still see?